packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -109,7 +109,6 @@ function MainLayoutHeader({
     return (
       <>
         <FeedNav />
-        <RenderSearchPanel />
```

Comment:
Unless anyone knows otherwise, but when it's not laptop, we only show the search pannel on search page

Reply:
Isn't this for tablet size?

Reply:
Nope, on tablet size we still don't show the search panel at all unless it is explore page

Reply:
Yeah I think that's where it was needed, something with explore on a specific size no?

Reply:
<img width="983" alt="Screenshot 2024-10-18 at 12 38 13 PM" src="https://github.com/user-attachments/assets/2c18506a-c94e-4d42-a14a-511e90c879df">
Based on this it's hidden if it's not explore page (the new search page).

Reply:
Hmm I've tried some options couldn't break anything, so guess it's good to go?

Reply:
I think so, code wise wether it falls into one block or another if it isSearchPage then it is most definitely rendered, if isSearchPage is false, then we were rendering, but then the css above was stoping it from showing.. so no point for it to get rendered. 

<img width="740" alt="Screenshot 2024-10-18 at 12 44 06 PM" src="https://github.com/user-attachments/assets/50e3b0db-7841-4ac1-81dc-1235b5e42e98">